src/modules/ItemSheet.ts

```diff
@@ -1,11 +1,15 @@
-import { i18n } from '../init';
 import { MystifiedData, MystifiedFlags } from './ForienUnidentifiedItemsModels';
 import Identification from './Identification';
-import { FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME, getGame } from './settings';
+import { i18n } from './lib/lib';
+import { FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME } from './settings';
+import { canvas, game } from './settings';
 
 export default function registerDerivedItemSheetClass() {
+  //@ts-ignore
   for (const k in CONFIG.Item.sheetClasses) {
+    //@ts-ignore
     for (const l in CONFIG.Item.sheetClasses[k]) {
+      //@ts-ignore
```

Comment:
Why were these `ts-ignore`'s needed?

Reply:
it's seems they are not present anymore on version 9 of foundry, i left these old reference for some retrocompatibility issue, maybe i can try to remove these...